app/src/main/java/memphis/myapplication/UI/ToolbarHelper.java

```diff
@@ -0,0 +1,41 @@
+package memphis.myapplication.UI;
+
+import androidx.appcompat.widget.Toolbar;
+
+import android.net.Uri;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.MemoryPolicy;
+import com.squareup.picasso.Picasso;
+
+import memphis.myapplication.R;
+import timber.log.Timber;
+
+public class ToolbarHelper {
+
+    private String title;
+    private View view;
+
+    public ToolbarHelper(String title, View view) {
+        this.title = title;
+        this.view = view;
+    }
+
+    public Toolbar setupToolbar(Uri uri) {
+        Timber.i("Got avatar uri: "+uri);
```

Comment:
Should delete this from the final code so it doesn't appear in the logs every time we switch fragments in the app.